rollup/rollup.config.js

```diff
@@ -0,0 +1,33 @@
+import commonjs from 'rollup-plugin-commonjs';
+import json from 'rollup-plugin-json';
+import resolve from 'rollup-plugin-node-resolve';
+import { terser } from 'rollup-plugin-terser';
+import babel from 'rollup-plugin-babel';
+
+const config = (name, path, output, filename) => ({
```

Comment:
Did a previous iteration necessitate the separation of `path` and `filename`? Seems like the usage might be cleaner just as `name, input, output`?